src/web/releases.rs

```diff
@@ -530,7 +530,9 @@ pub fn search_handler(req: &mut Request) -> IronResult<Response> {
                      FROM crates
                      INNER JOIN releases
                          ON crates.latest_version_id = releases.id
-                     WHERE github_stars >= 100 AND rustdoc_status = true
+                     LEFT JOIN github_repos
+                         ON releases.github_repo = github_repos.id
+                     WHERE github_repos.stars >= 100 AND rustdoc_status = true
```

Comment:
The `unwrap()` on line 540 panics if there are no results, but I think that's an existing bug that shouldn't happen in production because we have lots of crates with over 100 stars. Looks good otherwise.